plans/tier1.fmf

```diff
@@ -240,7 +240,14 @@ adjust+:
             test+<:
                 - conversion-method/activation_key_conversion
 
-    /firewalld-disabled-ol8:
+
+    /excluded-packages-removed:
+        discover+:
+            test+<:
+                - excluded-packages-removed/excluded_pkgs_removed
```

Comment:
```suggestion
    /yum_conf_exclude_packages:
        discover+:
            test+<:
                - yum-conf-exclude-packages/yum_conf_exclude_packages
```
I suppose that's a copy&paste error

Reply:
I renamed the test, but forgot to change it everywhere